deploy.yml

```diff
@@ -523,8 +523,8 @@ jobs:
       TF_VAR_vpc_id: '((preprod_vpc_id))'
       TF_VAR_ecs_cluster_name: 'preprod-eq'
       TF_VAR_docker_registry: {{docker_registry}}
-      TF_VAR_aws_alb_arn: {{preprod_aws_alb_arn}}
-      TF_VAR_aws_alb_listener_arn: {{preprod_aws_alb_listener_arn}}
+      TF_VAR_aws_alb_arn: {{preprod_internal_aws_alb_arn}}
+      TF_VAR_aws_alb_listener_arn: {{preprod_internal_aws_alb_listener_arn}}
```

Comment:
Where does the internal alb get created?
Hint: https://github.com/ONSdigital/eq-deploy/pull/46/files#diff-c5a2c990f03bcdb3e6d4e81cf49d64f5R299

Reply:
Fixed.